.github/workflows/test-phpunit.yml

```diff
@@ -4,12 +4,7 @@ on:
   push:
     branches:
       - develop
-      - '4.*'
     paths:
-      - 'app/**'
-      - 'public/**'
-      - 'system/**'
-      - 'tests/**'
```

Comment:
Why do you remove these paths?

Reply:
Because of the `'**.php'` below. It essentially covers them